Remotion/Data/DomainObjects.Web.Test/Program.cs

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Collections;
+using CoreForms.Web.Infrastructure;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Remotion.Data.DomainObjects.Web.Test
+{
+  public class Program
+  {
+    public static void Main (string[] args)
+    {
+      string virtualPath;
+      if (args.Length == 1)
+        virtualPath = args[0];
+      else
+        virtualPath = "/";
+
+      LegacyAspNetInitialization.License = "I hereby confirm that I use CoreForms only for trial purposes and have read and accept the CoreForms Trial License.";
+
+      var legacyAspNetInitializationOptions = new LegacyAspNetInitializationOptions(virtualPath: virtualPath, physicalPath: Environment.CurrentDirectory);
+      LegacyAspNetInitialization.Initialize(legacyAspNetInitializationOptions);
+
+      var webControls = (Hashtable)System.Web.Configuration.WebConfigurationManager.GetSection("system.web/webControls");
+      webControls["clientScriptsLocation"] = virtualPath.TrimEnd('/') + webControls["clientScriptsLocation"];
+
```

Comment:
Why do we need this magic? It feels like we need to do something here because that looks a bit hacky 😅

Reply:
That's me and I and to make it so I can specify the virtual directory of the application from outside . That way, I can host below root as well as in root. I was hoping to get it completed before now, alas...

This change will go into all the applications. If you want you can strip it out for now and keep it in a separate PR that does hosting cleanup for all applications.

Reply:
Removed it for now, added a new issue